supabase/functions/_backend/triggers/logsnag_insights.ts

```
@@ -269,6 +274,12 @@ async function getGithubStars(): Promise<number> {
 async function getBuildStats(c: Context): Promise<BuildStats> {
   const supabase = supabaseAdmin(c)
   const last30days = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000).toISOString()
+  const today = new Date()
+  const todayStartMillis = Date.UTC(today.getUTCFullYear(), today.getUTCMonth(), today.getUTCDate())
+  const prevDayStart = new Date(todayStartMillis - 24 * 60 * 60 * 1000)
+  const prevDayEnd = new Date(todayStartMillis)
+  const date_id = getDateId(today)
```

ESLint

[error] 'date_id' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)